Plots/Polygons/NCL_polyg_2.py

```diff
@@ -0,0 +1,140 @@
+"""
+NCL_polyg_2.py
+==============
+Concepts illustrated:
+  - Drawing a Lambert Conformal U.S. map color-coded by climate divisions
+  - Color-coding climate divisions based on precipitation values
+  - Drawing the climate divisions of the U.S.
+  - Zooming in on a particular area on a Lambert Conformal map
+  - Drawing a border around filled polygons
+  - Masking the ocean in a map plot
+  - Masking land in a map plot
+  - Increasing the font size of text
+  - Adding text to a plot
+  - Drawing a custom labelbar on a map
+  - Creating a red-yellow-blue color map
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/polyg_2.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/polyg_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+# ----------------
```

Comment:
Please remove this line to keep this heading consistent with other cell headings throughout the html page.